examples/ex1parametric.cpp

```diff
@@ -0,0 +1,427 @@
+//               libROM MFEM Example 1 parametric ROM Laplace problem 
+//
+// Compile with: ./scripts/compile.sh -m 
+//
+// Description:  This example code demonstrates the use of MFEM and libROM to
+//
+//               define a simple projection-based reduced order model of the
+//               Laplace problem -Delta u = f(x) with homogeneous Dirichlet
+//               boundary conditions and spatially varying right hand side f.  
+//
+//               The example highlights the three distinct processes, i.e.,
+//               offline, merge, and online. The offline phase runs MFEM full
+//               order model and store the snapshot data in HDF file. You can
+//               run as many offline phases as you wish to sample the parameter
+//               space.  The merge phase reads all the snapshot files, builds a
+//               global reduced basis, and stores the basis in a HDF file.  The
+//               online phase reads the basis, builds ROM operator, and solves
+//               reduced order system.
+//
+// Offline phase: ex1parametric -offline -f 1.0 -id 0
```

Comment:
This test works in parallel, e.g. with `mpirun -np 4`, so it may be helpful to add that here so the user knows it will work in parallel.